packages/ses/src/lockdown-shim.js

```diff
@@ -130,19 +143,27 @@ export const repairIntrinsics = (
   // for an explanation.
   options = /** @type {LockdownOptions} */ ({ ...firstOptions, ...options });
   const {
+    errorTaming = getenv('LOCKDOWN_ERROR_TAMING', 'safe'),
+    errorTrapping = getenv('LOCKDOWN_ERROR_TRAPPING', 'platform'),
+    regExpTaming = getenv('LOCKDOWN_REGEXP_TAMING', 'safe'),
+    localeTaming = getenv('LOCKDOWN_LOCALE_TAMING', 'safe'),
+    consoleTaming = getenv('LOCKDOWN_CONSOLE_TAMING', 'safe'),
+    overrideTaming = getenv('LOCKDOWN_OVERRIDE_TAMING', 'moderate'),
+    stackFiltering = getenv('LOCKDOWN_STACK_FILTERING', 'concise'),
+    domainTaming = getenv('LOCKDOWN_DOMAIN_TAMING', 'unsafe'),
```

Comment:
Isn't this now safe? Did this accidentally get reverted as merge damage?

Reply:
It’s still set to unsafe by default, gated by a release with breaking changes. This was released in a non-breaking change on Friday.

Reply:
The comment to that effect did get lost in merge shuffle though!